index.html

```diff
@@ -123,7 +123,8 @@ <h1>Tonic</h1>
                             <li>css</li>
```

Comment:
You are missing to add your linters for JS.
- Kindly add the required[ Linters](https://github.com/microverseinc/linters-config/tree/master/javascript) for this activity.